addons/web/static/src/views/form/form_controller.js

```diff
@@ -120,7 +122,9 @@ export class FormController extends Component {
         ButtonBox,
         ViewButton,
         Field,
-        CogMenu,
+        CogMenu: FormCogMenu,
+        StatusBarDropdownItems,
+        Widget,
```

Comment:
not used ?

Reply:
Actually, yes, it's used via the "magic" of the generated slots' template.